libopae/plugins/xfpga/sysfs.c

```diff
@@ -89,8 +91,11 @@ static sysfs_formats sysfs_path_table[OPAE_KERNEL_DRIVERS] = {
 	 .sysfs_compat_id = "/dfl-fme-region.*/fpga_region/region*/compat_id",
 	 .sysfs_fme_temp_glob = "hwmon/hwmon*/temp*_*",
 	 .sysfs_fme_pwr_glob = "hwmon/hwmon*/power*_*",
+	 .sysfs_fme_perf_glob = "*perf",
 	 .sysfs_port_err = "errors/errors",
-	 .sysfs_port_err_clear = "errors/errors"
+	 .sysfs_port_err_clear = "errors/errors",
+	 .sysfs_bmc_glob = "avmmi-bmc.*/bmc_info",
+	 .sysfs_max10_glob = "spi-*/spi_master/spi*/spi*.*"
```

Comment:
does the dfl driver export sysfs nodes for spi?

Reply:
DFL Driver doesn't support now, added to fix mock test cases.

Reply:
Should they be null then?